### DISCUSSION TITLE
sempre zigbee2mqtt

### DISCUSSION THREAD

**Discussion by filpao** [indomus/forum#1193] [ZigBee, Z-Wave, Thread, Matter]
Buonasera a tutti.
Dopo le vicissitudini con zigbee2mqtt accoppiato alle vecchie antenne 2530, ho deciso di passare a un hardware più moderno, e cioè Zbee-Duo.

Ha funzionato tutto alla perfezione fino ad oggi, quando di colpo tutti e 3 i dispositivi che ho hanno cominciato a rifiutare di parlare con il sistema.
L'errore che si vede nel log di zigbee2mqtt è il seguente:

`Zigbee2MQTT:error 2021-12-15 20:40:46: Publish 'set' 'state' to 'sw_luce_ingressoCasa' failed: 'Error: Command 0x0004740000b5f611/1 genOnOff.off({}, {"sendWhenActive":false,"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Data request failed with error: 'No network route' (205))'`

Questo è l'errore generato cercando di comandare un interruttore a muro bTicino da home assistant; i due sensori di movimento non hanno miglior sorte. In particolare mi insospettisce quel "no network route".

Piattaforma: Hassio su macchina virtuale VMware, con OS 6.x e poi 7 (funzionavano entrambi fino a oggi), Core 2021.11.x (varie releases, funzionavano tutte fino ad oggi), zigbee2mqtt 1.22.0-3; dopo il problema ho provato ad aggiornarlo a 1.22.1-1, ma niente da fare; provato anche a tornare alle vecchie versioni del Core e dell'OS (prima di fare un upgrade, prendo sempre uno snapshot a livello di VM), ma nulla da fare.

L'unica cosa che è cambiata rispetto a ieri, è che oggi Windows 10 ha fatto l'upgrade mensile, ma non capisco come questo possa aver influenzato una VM totalmente indipendente...

Qualche idea?

Ciao
Filippo

**Comment by snakuzzo** [comment]
Non stai comunicando con la chiavetta...da questo log più di questo non si capisce...mi spiace.
Se usi ser2net prova a fare dall'host

```
telnet ipchiavetta:porta
```

e vedi se risponde.
Se la usi via USB sinceramente non saprei che dirti oltre ad indagare sulla configurazione della tua VM e del tuo host
